core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -172,6 +172,11 @@ oppia.controller('ExplorationEditor', [
     var _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewTab';
     var _ID_TUTORIAL_SAVE_BUTTON = '#tutorialSaveButton';
 
+    var screenWidth = windowDimensionsService.getWidth();
+    if (screenWidth <= 1280) {
+      _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewSmall';
```

Comment:
Update [#1591] :
The last line changes from 
`windowDimensionsService.addOnResizeHook($scope.isNarrow);`
to
`windowDimensionsService.registerOnResizeHook($scope.isNarrow);`